sale_blanket_order/report/templates.xml

```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template id="report_blanketorder_document">
+        <t t-call="web.external_layout">
+            <t t-set="doc" t-value="doc.with_context({'lang':doc.partner_id.lang})" />
+            <div class="page">
+                <div class="oe_structure" />
+                <div class="row">
+                    <div class="col-xs-6">
+                        <div
+                            t-field="doc.partner_id"
+                            t-options='{"widget": "contact", "fields": ["address", "name", "phone", "fax"], "no_marker": True, "phone_icons": True}'
+                        />
+                        <p t-if="doc.partner_id.vat">VAT: <span
+                                t-field="doc.partner_id.vat"
+                            /></p>
+                    </div>
+                </div>
+                <h2>
+                    <span>Blanket Order # </span>
+                    <span t-field="doc.name" />
+                </h2>
+                <div class="row mt32 mb32" id="informations">
+                    <div t-if="doc.client_order_ref" class="col-3">
+                        <strong>Your Reference:</strong>
+                        <p t-field="doc.client_order_ref" />
+                    </div>
+                    <div class="col-3">
+                        <strong>Validity Date:</strong>
+                        <p t-field="doc.validity_date" />
+                    </div>
+                    <div t-if="doc.user_id.name" class="col-3">
+                        <strong>Salesperson:</strong>
+                        <p t-field="doc.user_id" />
+                    </div>
+                    <div t-if="doc.currency_id" class="col-3">
+                        <strong>Currency:</strong>
+                        <p t-field="doc.currency_id" />
+                    </div>
+                </div>
+                <table class="table table-condensed">
+                    <thead>
+                        <tr>
+                            <th>Product</th>
+                            <th class="text-right">Unit Price</th>
```

Comment:
In version 16 the css classes text-right and text-left has been renamed to text-end and text-start